src/Sfa.Roatp.Indexer.ApplicationServices/ProviderIndexer.cs

```diff
@@ -144,15 +144,10 @@ public IEnumerable<RoatpProvider> LoadEntries()
             return _providerDataService.GetRoatpData();
         }
 
-        public async Task IndexEntries(string indexName, IEnumerable<RoatpProvider> roatpProviders)
+        public void IndexEntries(string indexName, IEnumerable<RoatpProvider> roatpProviders)
```

Comment:
In the calling method, the IEnumerable<RoatpProvider> is cast to a list - would it make sense to make that the same here?